eventkitd-bigquery/bigquery/client.go

```diff
@@ -0,0 +1,214 @@
+package bigquery
+
+import (
+	"cloud.google.com/go/bigquery"
+	"context"
+	"github.com/jtolio/eventkit/pb"
+	"github.com/pkg/errors"
+	"google.golang.org/api/googleapi"
```

Comment:
import order? i know we don't have linting in this repo

Reply:
We have linting in this repo, just we don't care about the import order.

But looks like you would prefer it ;-).  Here is the patch to enable:

https://github.com/storj/eventkit/pull/21